src/PVRIptvData.cpp

```diff
@@ -334,11 +357,12 @@ bool PVRIptvData::LoadEPG(time_t iStart, time_t iEnd)
     GetNodeValue(pChannelNode, "category", entry.strGenreString);
```

Comment:
you should do the same for category

Reply:
That depends, if someone uses a mapping file for genres it would break functionality. So would need to be added as a setting.

Reply:
Settings changes I think I would leave for Matrix branch, as well as many of the other xmltv fields.